modules/utiqMtpIdSystem.md

```diff
@@ -0,0 +1,15 @@
+## Utiq User ID Submodule
+
+Utiq MTP ID Module.
+
+First, make sure to add the utiq MTP submodule to your Prebid.js package with:
```

Comment:
This submodule seems to require additional setup by publishers to get the utiqMtpId placed in local storage. That extra installation should be made clear in the docs here as well as in the documentation repo.

Reply:
No there isn't any extra setup. Utiq script (external to prebid) that is installed separately by publisher already save the id in localstorage. The prebid module simply use prebid localstorage api to access the value when available.

Reply:
so that's my point - if an external script is required for this to work, you should mention that in the documentation (pbjs and prebid docs) so that publishers are aware they can't just turn this on without registering with utiq and installing additional code on their site.

Reply:
ah sorry, I misunderstood your comment. Yes that makes perfect sense. I just added some extra information in the module documentation to notify publishers that some extra script configuration is required for the module to operates and provided link to utiq contact form if they are interested. Let me know if that address your point.